qualtran/bloqs/rotations/hamming_weight_phasing.py

```diff
@@ -94,6 +94,20 @@ def build_call_graph(self, ssa: 'SympySymbolAllocator') -> Set['BloqCountT']:
         }
 
 
+@bloq_example
+def _hamming_weight_phasing() -> HammingWeightPhasing:
+    hamming_weight_phasing = HammingWeightPhasing(4, np.pi / 2.0)
+    print("Applying this unitary to |1111> should be the identity, and |0101> will flip the sign.")
+    return hamming_weight_phasing
+
+
+_HAMMING_WEIGHT_PHASING_DOC = BloqDocSpec(
+    bloq_cls=HammingWeightPhasing,
+    import_line='from qualtran.bloqs.rotations.hamming_weight_phasing import HammingWeightPhasing',
```

Comment:
the `import_line` argument should be unnecessary. The default should work in most cases including this one.